### PR TITLE
Clean up PaymentProcessor

### PR DESCRIPTION
**Pull request by rkondratowicz**
- clearer param name
- more logging
- formatting

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java [hunk 1]
```diff
@@ -44,12 +43,15 @@ public void createPayments(Envelope envelope, Long ccdId) {
 
             LOG.info("Started processing payments for case with CCD reference {}", cmd.ccdReference);
             paymentsPublisher.send(cmd);
-            LOG.info("Finished processing payments for case with CCD reference {}", cmd.ccdReference);
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java]: Why not have this log entry?

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java [hunk 2]
```diff
@@ -44,12 +43,15 @@ public void createPayments(Envelope envelope, Long ccdId) {
...
+        } else {
+            LOG.info(
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java]: 👍

**Comment by lgonczar** [comment]
As this is quite a generic PR, maybe we could also rename `LOG` to `log`, as per convention?

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java (hunk 1 above)
Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java]: reverted

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java [hunk 3]
```diff
@@ -18,21 +18,20 @@
 
 @Service
 public class PaymentsProcessor {
-    private static final Logger LOG = LoggerFactory.getLogger(PaymentsProcessor.class);
+
+    private static final Logger log = LoggerFactory.getLogger(PaymentsProcessor.class);
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java]: Codacy Issue found: [The constant name 'log' doesn't match '[A-Z][A-Z_0-9]*'](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4543433)

**Comment by lgonczar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/bulk-scan-orchestrator/bulk-scan-orchestrator/pullRequest?prid=4543433&bid=15289045)

**Pull request merged**